website_medical/static/src/js/file_upload_text_insert.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 LasLabs Inc.
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+$(document).ready(function() {
```

Comment:
Yes, please remove this file

Example of correct onload though https://github.com/OCA/maintainer-tools/blob/master/template/module/static/src/js/website_module_name.js